### DISCUSSION TITLE
Information on supported etcd releases

### DISCUSSION THREAD

**Discussion by pchan** [etcd-io/etcd#15475] [General]
I am trying to locate information on supported etcd releases in upstream documentation. Is it 2 minor versions at any point of time ? 

The etcd 3.6 release is still in [alpha](https://github.com/etcd-io/etcd/issues/13538), so can I assume that version 3.5 will be supported until at-least 3.7 be released ? 

Also, is there any tentative date when 3.6 release can be expected, there has been no update in the 3.6 issue for some time. I have an issue that refactors the release process ([url](https://github.com/etcd-io/etcd/issues/15396)). I would like to target that for 3.6.

cc: @serathius

**Comment by ahrtr** [comment]
Please read https://github.com/etcd-io/website/pull/601 and https://etcd.io/docs/v3.5/op-guide/versioning/
